internal/config/config.go

```diff
@@ -1,5 +1,52 @@
 package config
 
+import (
+	"emperror.dev/errors"
+	"github.com/spf13/viper"
+)
+
 // Version is the version of the av application.
 // It is set automatically when creating release builds.
 var Version = "<dev>"
+
+var GitHub struct {
+	Token string
+}
+
+// Load initializes the configuration values.
+// It may optionally be called with a list of additional paths to check for the
+// config file.
+// Returns a boolean indicating whether or not a config file was loaded and an
+// error if one occurred.
+func Load(paths []string) (bool, error) {
+	config := viper.New()
+
+	// Viper has support for various formats, so it supports kson, toml, yaml,
+	// and more (https://github.com/spf13/viper#reading-config-files).
+	config.SetConfigName("config")
+
+	// Reasonable places to look for config files.
+	config.AddConfigPath("$XDG_CONFIG_HOME/av")
```

Comment:
what's this? `$XDG_CONFIG_HOME`

Reply:
also this should probably be `.av` as well.

Reply:
It's just a standard for where people put configuration if it's defined. It shouldn't be `.av` since it's already inside a config directory.